projects/packages/my-jetpack/_inc/data/products/use-products-by-ownership.ts

```diff
@@ -0,0 +1,42 @@
+import { REST_API_SITE_PRODUCTS_OWNERSHIP_ENDPOINT } from '../constants';
+import { QUERY_PRODUCT_BY_OWNERSHIP_KEY } from '../constants';
+import useSimpleQuery from '../use-simple-query';
+// import { getMyJetpackWindowInitialState } from '../utils/get-my-jetpack-window-state';
+import type { ProductSnakeCase } from '../types';
+// import type { RefetchOptions, QueryObserverResult } from '@tanstack/react-query';
+
+// Create query to fetch new product data from the server
+const useFetchProductsByOwnership = () => {
+	const queryResult = useSimpleQuery< ProductSnakeCase >( {
```

Comment:
I think the expected return type of this endpoint  should be`JetpackModules[]`, or maybe `Record<'owned'|'unowned', JetpackModules[]`?